admin/services/data-access/sql.service.js

```diff
@@ -40,13 +87,15 @@ sqlService.query = (sql, params) => {
     }
     let results = []
     // http://tediousjs.github.io/tedious/api-request.html
+    winston.debug(`sql.service: SQL: ${sql}`)
```

Comment:
we can add a new category in winston.  Would it be better to do this now? 🤔

Reply:
I think `debug` is fine, but I don't object to a new category.